frontend/src/modules/search/components/FiltersPanel.js

```diff
@@ -84,6 +252,16 @@ export class FiltersPanelBase extends React.Component<Props, State> {
         };
     }
 
+    createApplyTimeRangeFilter() {
+        const { chartFrom, chartTo, isChartVisible } = this.state;
+
+        if (isChartVisible) {
+            return;
+        }
+
+        return this.createApplySingleFilter([chartFrom, chartTo].join('-'), 'timeRange');
+    }
```

Comment:
The same goes here, just make a simple method that will be called directly, no need to create a closure.